src/ImageSharp/Processing/Processors/Transforms/Resize/ResizeHelper.cs

```diff
@@ -72,7 +72,7 @@ public static (Size, Rectangle) CalculateTargetLocationAndBounds(Size sourceSize
 
                 // case ResizeMode.Stretch:
                 default:
-                    return (new Size(width, height), new Rectangle(0, 0, width, height));
+                    return (new Size(Sanitize(width), Sanitize(height)), new Rectangle(0, 0, width, height));
```

Comment:
Do we not need to sanatize the second width and height?

Reply:
Yeah, missed that with my find/replace